src/components/themed/TransactionListTop.tsx

```diff
@@ -85,12 +86,14 @@ interface StateProps {
   balanceMap: EdgeBalanceMap
   currencyCode: string
   defaultFiat: string
+  dispatch: Dispatch
```

Comment:
This is a bit unconventional - normally we use `mapDispatchToProps` for this situation, but I can't see any reason why it's bad, so yay for finding simple solutions. 👍

Reply:
I originally was passing in callbacks for each dispatch, then realized the redundancy. As this component is slowly transitioned into a function-component, then the prop goes away.